linkchecker

```diff
@@ -331,7 +331,7 @@ group.add_argument("--ignore-url", action="append", metavar="REGEX",
 """Only check syntax of URLs matching the given regular expression.
  This option can be given multiple times."""))
 group.add_argument("--ignore-class", action="append",
```

Comment:
Maybe 
```suggestion
group.add_argument("--ignore-classes", action="append",
```
?

argparse allows the user to shorten long option names, so --ignore-class will continue to be allowed.